### PR TITLE
Prev/Next Button Styling

### PR DESCRIPTION
**Pull request by joshdarby**
Before:
Screen Shot 2019-06-03 at 3 11 50 PM

After:
Screen Shot 2019-06-03 at 3 11 32 PM

Based on issue #75 with mockup in #76.

### CODE REVIEW

**Comment by benlk** [comment]
Side-by-side arrow and text is fine on wider viewports, but on narrower viewports would it make sense to have the arrow and text be on separate lines? 
Screen Shot 2019-06-03 at 16 39 22

File: wp-content/themes/cjet/guides.php [hunk 1]
```diff
@@ -137,14 +149,14 @@
 							);
 						} else {
 							if (!empty($prev_id)) {
-								printf( '<div class="previous"><a href="%1$s"><h5>Previous Section</h5><span class="meta-nav">%2$s</span></a></div>',
+								printf( '<div class="previous"><a href="%1$s"><i class="dashicons dashicons-arrow-left-alt"></i><span class="meta-nav">%2$s</span></a></div>',
```
Comment (benlk) [inline wp-content/themes/cjet/guides.php]: should this `<a` be `rel="prev"`?

https://www.w3.org/TR/WCAG20-TECHS/H59.html describes why, https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types says that the `rel=""` attribute can be used on `<a>`.

File: wp-content/themes/cjet/guides.php [hunk 2]
```diff
@@ -137,14 +149,14 @@
...
 									get_permalink( $prev_id ),
 									get_the_title($prev_id)
 								);
 							}
 
 							if (!empty($next_id)) {
-								printf( '<div class="next"><a href="%1$s"><h5>Next Section</h5><span class="meta-nav">%2$s</span></a></div>',
+								printf( '<div class="next"><a href="%1$s"><i class="dashicons dashicons-arrow-right-alt"></i><span class="meta-nav">%2$s</span></a></div>',
```
Comment (benlk) [inline wp-content/themes/cjet/guides.php]: And if we mark the previous link as `rel="prev"`, this should be `rel="next"`.

**Comment by joshdarby** [comment]
@benlk Yea, I think that would make sense. I'll go ahead and implement it like that. 

Which guide are you seeing that on? All of the ones I looked at fit fine in the 50/50 space, so just want to check this one out too.

**Comment by benlk** [comment]
One that might not exist in your db; it's https://learn.inndev.test/guides/business-resources/tools-nonprofit-news/, post ID `3054`

File: wp-content/themes/cjet/guides.php (hunk 1 above)
Comment (joshdarby) [reply wp-content/themes/cjet/guides.php]: Good call.

**Pull request merged**